vllm/v1/metrics/stats.py

```diff
@@ -57,14 +57,17 @@ def update_from_output(self, output: "EngineCoreOutput",
 
         self.num_generation_tokens += num_new_generation_tokens
         if is_prefilling:
-            # This relies on the invariant that EngineCore does
-            # not stream outputs for partially completed prefills
-            # (scheduler.update_from_output makes EngineCoreOutput
-            # iff num_computed_tokens == num_tokens).
-            assert (num_new_generation_tokens > 0)
-            self.num_prompt_tokens += prompt_len
-
-            self.time_to_first_tokens_iter.append(last_token_latency)
+            # TODO(andy): we used to assert that num_new_generation_tokens
```

Comment:
cc @markmc - FYI. We will revert this in a follow up. But just wanted to give you an FYI.